aten/src/ATen/ceil_div.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include <c10/macros/Macros.h>
+
+namespace at {
+
+/**
+   Computes ceil(a / b)
+*/
+template <typename T, typename=std::enable_if_t<std::is_integral<T>::value>>
+C10_ALWAYS_INLINE C10_HOST_DEVICE T ceil_div(T a, T b) {
```

Comment:
There's already ATenCeilDiv in CUDAApplyUtils.cuh already, can you use it? And add `round_up` to that header.

Reply:
I've done it the other way around, replacing ATenCeilDiv with `ceil_div`. My reasoning is that this isn't cuda specific, and would be useful to have available for future CPU code. Also, that including all of `CUDAApplyUtils.cuh` just for a small helper function is unnecessary.